Resources/views/Calendar/view.html.twig

```diff
@@ -119,7 +119,15 @@
                             <hr/>
                         </div>
                         {% endif %}
-                        <div><label class="bold">{{'global.stop_point_code'|trans({}, 'messages')}}</label>: {{ calendar.schedules.stop_point.codes|externalCode }}</div>
+                        <div>
+                            <dl>
+                                <dt>{{'global.stop_point_code'|trans({}, 'messages')}}</dt>
+                                {% for code in calendar.schedules.stop_point.codes %}
+                                {% set value = code.type == 'external_code' ? code.value|slice(3) : code.value %}
```

Comment:
Ideally, you should have ViewModel objects where you prepare such stuff as removing digits, preformat values with defaults, and so on, and so forth. This will:
* Remove the logic from the view
* Decouple the view logic from the actual model (here Navitia).

But this means some refactoring. Are you ready to do it in that sprint?